### PR TITLE
fix(DB/SmartScript) Sundered Rumblers wrong use of Summon Sundered Shard

### PR DESCRIPTION
**Pull request by M4th3m4tic4l**
## Changes Proposed:
-  Casts Summon Sundered Shard on death
-  Says text "Sundered Rumbler shatters into shards."

## Issues Addressed:

- https://github.com/azerothcore/azerothcore-wotlk/issues/14818
- https://github.com/chromiecraft/chromiecraft/issues/4908

## SOURCE:

https://youtu.be/HX8haCDIVT0?t=165

## Tests Performed:

- 
WoWScrnShot_013123_181633

## How to Test the Changes:

1. Insert the changes in your world database
2. .go c 67708
3. .damage 50000

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Tested and work fine. Now he cast "Summon Sundered Shard" only on death and emote is present.

File: data/sql/updates/pending_db_world/rev_1675207681546911800.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+--
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 18881;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(18881,0,0,"Sundered Rumbler shatters into shards.",16,0,100,0,0,0,0,0,"Sundered Rumbler - Emote");
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1675207681546911800.sql:5]: Missing broadcasttextId

You can find it by searching the broadcast_text table

```sql
SELECT * FROM `broadcast_text` WHERE `MaleText` LIKE "%shatters into shards%";
```

Or

```sql
SELECT * FROM `broadcast_text` WHERE `FemaleText` LIKE "%shatters into shards%";
```

**Review by Nyeriah — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1675207681546911800.sql (hunk 1 above)
Comment (Nyeriah) [reply data/sql/updates/pending_db_world/rev_1675207681546911800.sql:5]: image

Should be 19115

**Review by Nyeriah — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1675207681546911800.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+--
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 18881;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(18881,0,0,"Sundered Rumbler shatters into shards.",16,0,100,0,0,0,19115,0,"Sundered Rumbler - Emote");
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1675207681546911800.sql:5]:
```suggestion
(18881,0,0,"%s shatters into shards.",16,0,100,0,0,0,19115,0,"Sundered Rumbler - Emote");
```

**Pull request merged**